### PR TITLE
Issue #993 stuck conditions

### PR DESCRIPTION
**Pull request by perkinss**
*Issue #: #993 *

*Description of changes:*

- Clear the  condition when the examiner accepts the name.  This way it is gone when they go to getNext, or go back and examine the next name, but it remains when they go back without accepting.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by perkinss** [comment]
Coming: tests

**Comment by katiemcgoff** [comment]
I'm reviewing...

File: client/src/components/application/Examine/CompName.vue [hunk 1]
```diff
@@ -363,6 +363,7 @@
       },
       nameAccept() {
         this.$store.commit('decision_made', 'APPROVED');
+        this.$store.commit('currentCondition', null);
```
Comment (katiemcgoff) [inline client/src/components/application/Examine/CompName.vue]: nice! can you also include this in the nameReject(), so it clears out in that scenario as well - there is a minimal edge case where it is logical to reference conditions so it does make sense, but also we don't prevent the user from having conditions selected even when rejecting (even if it doesn't make sense) so might as well put it in for good measure.

Comment (perkinss) [reply client/src/components/application/Examine/CompName.vue]: ah yes, thanks!

**Comment by perkinss** [comment]
I added the suggested change

**Pull request merged**